src/test/java/io/aiven/kafka/connect/transforms/ExtractTopicTest.java

```diff
@@ -139,10 +189,24 @@ void fieldName_UnsupportedTypeInField(final boolean skipMissingOrNull) {
                 + " must be [INT8, INT16, INT32, INT64, FLOAT32, FLOAT64, BOOLEAN, STRING]: " + originalRecord);
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void fieldName_Schemaless_UnsupportedSchemaTypeInField(final boolean skipMissingOrNull) {
```

Comment:
I'd prefer camel case names.

Reply:
me too, but was following the existing convention on this class. If you feel strongly, I can refactor the whole class.

Reply:
Maybe a cleanup PR for the project later on.